hasher-matcher-actioner/scripts/submitter.py

```diff
@@ -0,0 +1,105 @@
+#! /usr/bin/env python3
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import os
+import time
+import threading
+import uuid
+import datetime
+import typing as t
+from submit_content_test import DeployedInstanceTestHelper
+
+
+class Submitter(threading.Thread):
+    def __init__(
+        self,
+        helper: DeployedInstanceTestHelper,
+        batch_size: int,
+        seconds_between_batches: int,
+        filepaths: t.List[str] = [],
+        **kwargs,
+    ):
+        super(Submitter, self).__init__(**kwargs)
+        self.daemon = True
+        self._stop_signal = threading.Event()
+        self._lock = threading.Lock()
+        self.helper = helper
+        self.batch_size = batch_size
+        self.seconds_between_batches = seconds_between_batches
+        self.filepaths = filepaths
+        self.total_submitted = 0
+
+    def stop(self):
+        self._stop_signal.set()
+
+    def stopped(self):
+        return self._stop_signal.is_set()
+
+    def run(self):
+        while not self.stopped() and self._lock.acquire():
+            if self.stopped():
+                self._lock.release()
+                return
+            try:
+                batch_prefix = f"soak-test-{str(uuid.uuid4())}"
+                for i in range(self.batch_size):
+                    content_id = (
+                        f"{datetime.datetime.now().isoformat()}-time-{batch_prefix}-{i}"
```

Comment:
Do we want to do clean up after running a soak test?

The choice of putting the datetime _before_ the `batch_prefix` makes it quite difficult to cleanup s3 and dynamodb.